lib/chef/knife/windows_cert_generate.rb

```diff
@@ -28,15 +28,19 @@ class WindowsCertGenerate < Knife
 
       banner "knife windows cert generate (options)"
 
-      option :domain,
-        :short => "-d DOMAIN",
-        :long => "--domain DOMAIN",
-        :description => "By default there will be no domain name. If user wants to give the hostname as '*.mydomain.com' then he must specify the domain as: --domain 'mydomain.com'"
+      option :hostname,
+        :short => "-H HOSTNAME",
+        :long => "--hostname HOSTNAME",
+        :description => "By default hostname is *. If user wants to give his fqdn as the hostname
+        then he must specify the hostname as: --hostname 'something.mydomain.com'. User can also give wildcard
+        as '*.mydomain.com'"
 
       option :output_file,
         :short => "-o PATH",
         :long => "--output-file PATH",
-        :description => "By default 3 files will be generated in the current folder as winrmcert.pfx and winrmcert.pem. You can specify alternate file path using this option. Eg: --output-file /home/.winrm/server_cert.pfx. This will create 3 files in the specified path with name server_cert.pem, sever_cert.pfx, server_cert.der.",
+        :description => "By default 3 files will be generated in the current folder as winrmcert.pfx and
+        winrmcert.pem. You can specify alternate file path using this option. Eg: --output-file /home/.winrm/server_cert.pfx.
+        This will create 3 files in the specified path with name server_cert.pem, sever_cert.pfx, server_cert.der.",
```

Comment:
If we normally output three files, what happens when I provide ```--output-file foo.pfx```? Does it only generate the pfx file? Or does this just change the prefix of the file, e.g. ```--output-file /tmp/foo``` would generate three files with three different extensions with a filename of foo in /tmp?

There are also some inconsistencies in the description:

* the list of files is provided twice
* the first list of files says there are three files but only lists two
* We don't generate .der files anymore, instead we chose .b64